testing-utils/src/lib/fixtures/configs/core-config.mock.ts

```diff
@@ -0,0 +1,73 @@
+import { CoreConfig } from '../../../../../packages/models/src';
+
+export const CORE_CONFIG_MOCK = {
```

Comment:
I would keep the "minimal" in the name, so that we distinguish it from the more realistic test fixtures we have in `models`.

Reply:
It is not really minimal as it has plugins and audits filled in. A minimal config would only have the required values of a minimum extent, so most arrays would be empty.

Reply:
So maybe we could name the others "example" (e.g. `EXAMPLE_REPORT`)?

Reply:
For the report, I will use `MINIMAL_REPORT` because it does only have minimum values. As for the others, once I get to `models` and other packages, I intend to unify as many of the configs as possible, so we will see then.